src/__tests__/__snapshots__/SearchResults.test.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This isnt a useful snapshot, you can only produce. one snapshot per file and it should represent the component as it appears when it loads initially. More details about this on the test file